client/landing/gutenboarding/onboarding-block/design-selector/style.scss

```diff
@@ -123,16 +123,22 @@ $deisgn-selector-selection-space: 175px;
 .page-layout-selector__grid {
 	display: grid;
 	grid-gap: 1.25em;
-	@include breakpoint( '>660px' ) {
-		grid-template-columns: 1fr 1fr 1fr;
-	}
+	grid-template-columns: repeat( auto-fill, minmax( 280px, 1fr ) );
 }
 
 .page-layout-selector__item {
 	position: relative;
 	overflow: hidden;
 	padding: 2px;
 
+	&:before {
+		content: "";
+		display: inline-block;
+		width: 1px;
+		height: 0;
+		padding-bottom: 87.5%;
```

Comment:
If these come from a known ratio, it can be nice to express that:

```
padding-bottom: ( 3 / 4 ); // not really the ratio
```

If we've just got a magic number (that seems to be the case), let's explain this:

```
padding-bottom: 87.5%; // This maintains a fixed aspect ration for the elements.
```

Reply:
Might be nice to explicitly `calc()`ulate that magic number from a given aspect ratio tho :grimacing:

Reply:
cc @dwolfe (can we get to a more natural ratio to satisfy the above concerns?)

Reply:
Well, 87.5 is just the calc'd result of 7/8, so we could use that. Is the ratio actually more clear than the result, though?

Reply:
I think so. It might still deserve an explanatory comment to hightlight that we're doing this for the sake of a fixed aspect ratio (rather than out of some weird percentage-based padding).

Reply:
Agreed, either way works. I don't actually have a strong opinion one way or the other, I was mostly just responding to "magic numbers", which I usually take to mean a fixed value with no explanation, ie. you can't tell how it was calculated. 7/8 and 87.5% are basically the same thing - they operate on the same starting value. That's all I wanted to call out.